TweetClassifier/src/main.py

```diff
@@ -178,6 +178,54 @@ def train_model_bert():
     print(model_outputs)
     print(wrong_prediction)
 
+    # TODO: Create retrain function
+
+def retrain_bert():
+    # 1. Get un-trained classified data
+    # 2. Retrieve BERT model
+    # 3. Train
+
+
+
+
+    db_contents = fetch()
+    ### CHECK ###
+    tweets = [x for x in db_contents if 'trained' not in x.keys()]
+    ### TESTING BELOW ###
+    for tweet in tweets:
+        tweet['trained'] = 1
+    collection = get_database_collection
```

Comment:
Missing parenthesis on the end of that function call

Reply:
(It is (done))